src/walletConnect/saga.ts

```diff
@@ -410,6 +410,14 @@ export function* normalizeTransaction(rawTx: any, network: Network) {
     tx.nonce = yield* call(getTransactionCount, publicClient[network], txCountParams)
   }
 
+  if ('chainId' in tx) {
```

Comment:
should we do this only for Celo, since the comment below says that this is only a problem there? i also wondered if this can become a whack-a-mole situation where there are more properties on the tx that could cause problems (should we keep an inclusion list of properties we actually want and delete all other properties?)

Reply:
Stripping it is ok on all chains.

Yes I've thought about this whack-a-mole situation, I was looking at the internals of viem as my initial thinking was that it should probably be handled there instead. I'll chat with the authors to see what they think.
Otherwise we can do an inclusion list ourselves, but I'd prefer if we could avoid it.